src/main/js/dynamic/DistributionConfiguration.js

```diff
@@ -223,14 +230,23 @@ class DistributionConfiguration extends Component {
                     <Modal.Body>
                         <form className="form-horizontal" onSubmit={this.handleSubmit} noValidate>
                             <FieldsPanel
-                                descriptorFields={currentChannel.fields}
+                                descriptorFields={commonFields}
+                                currentConfig={channelConfig}
+                                fieldErrors={this.props.fieldErrors}
+                                self={this}
+                                stateName="channelConfig"
+                            />
+                            {currentChannel && selectedProvider &&
+                            <FieldsPanel
+                                descriptorFields={channelFields}
```

Comment:
This change should always show the common fields and the channel and provider fields will only be shown when the channel type and provider have been selected.